tket/src/Mapping/LexiLabelling.cpp

```diff
@@ -0,0 +1,65 @@
+#include "Mapping/LexiLabelling.hpp"
```

Comment:
add the copy right note

Reply:
(in all files...?)

Reply:
This was done upstream and i've now pulled in. However updated now for new files!